cloud_governance/policy/aws/cost_billing_reports.py

```diff
@@ -0,0 +1,134 @@
+import datetime
+import os
+
+import pandas as pd
+
+from cloud_governance.common.clouds.aws.cost_explorer.cost_explorer_operations import CostExplorerOperations
+from cloud_governance.common.clouds.aws.iam.iam_operations import IAMOperations
+from cloud_governance.common.clouds.aws.sts.sts_oprations import STSOperations
+from cloud_governance.common.elasticsearch.elastic_upload import ElasticUpload
+from cloud_governance.common.google_drive.google_drive_operations import GoogleDriveOperations
+from cloud_governance.main.environment_variables import environment_variables
+
+
+class CostBillingReports:
+    """
+    This class is reposnsible for extarcting
```

Comment:
This class is responsible for generation cost billing report for Budget, Actual, Forecast